channels/facebook/src/main/kotlin/com/justai/jaicf/channel/facebook/FacebookChannel.kt

```diff
@@ -20,12 +22,13 @@ class FacebookChannel private constructor(
     InvocableBotChannel {
 
     private lateinit var messenger: Messenger
+    private var liveChatProvider: JaicpLiveChatProvider? = null
 
     constructor(botApi: BotApi, config: FacebookPageConfig) : this(botApi) {
         messenger = Messenger.create(config.pageAccessToken, config.appSecret, config.verifyToken)
     }
 
-    private constructor(botApi: BotApi, baseUrl: String) : this(botApi) {
+    private constructor(botApi: BotApi, baseUrl: String, liveChatProvider: JaicpLiveChatProvider) : this(botApi) {
         messenger = Messenger.create("", "", "", baseUrl)
```

Comment:
I guess here we need to assign liveChatProvider to the field

Reply:
Thanks! Fixed